model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/liquibase/updater/MapJpaLiquibaseUpdaterProvider.java

```diff
@@ -42,23 +42,6 @@
 
 public class MapJpaLiquibaseUpdaterProvider implements MapJpaUpdaterProvider {
 
-    public static class ThreadLocalSessionContext {
-
-        private static final ThreadLocal<KeycloakSession> currentSession = new ThreadLocal<KeycloakSession>();
-    
-        public static KeycloakSession getCurrentSession() {
-            return currentSession.get();
-        }
-    
-        public static void setCurrentSession(KeycloakSession session) {
-            currentSession.set(session);
-        }
-    
-        public static void removeCurrentSession() {
-            currentSession.remove();
-        }
-    }
-    
```

Comment:
+1 to remove this - this thread local context was copied over from the legacy provider and was used to supply the session to the custom changes and was never really needed in the new store.